projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -495,4 +607,22 @@ public static function get_cart_checkout_info() {
 
 		return $info;
 	}
+
+	/**
+	 * Save createaccount post data to be used in $this->order_process.
+	 *
+	 * @param array $data post data from the checkout page.
+	 *
+	 * @return array
+	 */
+	public function save_checkout_post_data( array $data ) {
+		$session = WC()->session;
+		if ( is_object( $session ) ) {
+			if ( isset( $data['createaccount'] ) && ! empty( $data['createaccount'] ) ) {
+				$session->set( 'wc_checkout_createaccount_used', true );
+				$session->save_data();
+			}
+		}
```

Comment:
Is this the best way to detect that a new account was requested? Maybe there's another hook fired when the account is created.